singolo.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+    <link rel = "stylesheet" href = "/singolo.css">
+    <title>Singolo</title>
+</head>
+<body>
+  <header class="main-navigation">
+    <div class="wrapper">
+      <nav class="navigation">
+        <div class="burger">
+          <div class="burger-menu-button">
+            <span class="burger-line"></span>
+          </div>
+        </div>
+        <a><p class="logo">Singolo<span class="symbol-logo">*</span></p></a>
+        <div class="navigation-menu">
+          <a href="#home" id="itemHome" class="navigation-menu-items item-home">home</a>
```

Comment:
It would be better to change classes like:
```suggestion
          <a href="#home" id="itemHome" class="navigation-menu-items navigation-menu-items-active">home</a>
```